CODEOWNERS

```diff
@@ -0,0 +1,120 @@
+# Code owner specification in Rucio
+#
+# Please consult the contribution guide for general contribution guidelines https://github.com/rucio/rucio/blob/master/CONTRIBUTING.rst
+#
+# In addition to peer-review, which is open to the entire Rucio community, we also notify the respective component leads to review pull requests involving their components.
+# Component leads: https://github.com/rucio/rucio/wiki
+
+# List of components:
+# Some components are out of the alphabetical order due to precedence of CODEOWNER files
+
+# Core & Internals (This goes first and is a catch-all for core)
+lib/rucio/core/                         @bari12
+lib/rucio/common/                       @bari12
+
+# Clients
+lib/rucio/client/baseclient.py          @mlassnig
+lib/rucio/client/uploadclient.py        @mlassnig
+lib/rucio/client/downloadclient.py      @mlassnig
+lub/rucio/client/                       @mlassnig
```

Comment:
-lub +lib ;)

Reply:
oh and then remove the other entries pointing to files inside `lib/rucio/client/` ?

Reply:
Will change; Right the other ones can be removed. 👍